docs/database-schema.md

```
@@ -0,0 +1,469 @@
+| `twitchOnlineNotification` | `id AUTO_INCREMENT` | Twitch stream online notifications |
+| `brawlstarsLinkedAccounts` | `user_id` | Linked Brawl Stars accounts |
+| `feedbackBlocked` | `user_id` | Users blocked from sending feedback |
+| `reportchannel` | `(guild_id, channel_id)` | Report submission channels |
+
+---
+
+## Table Relationships
+
+### Foreign Key Relationships
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)